app/services/usps_confirmation_maker.rb

```diff
@@ -10,8 +10,7 @@ def otp
   end
 
   def perform
-    entry = UspsConfirmationEntry.new_from_hash(attributes)
-    UspsConfirmation.create!(entry: entry.encrypted)
+    UspsConfirmation.create!(entry: attributes.to_json)
```

Comment:
I think we still want this to be encrypted, right? This will result in raw PII in our database otherwise.

Reply:
Just saw your comment, above. Yeah, the PII in temporarily saved in the DB needs to be encrypted.